subprojects/docs-asciidoctor-extensions-base/src/main/java/org/gradle/docs/asciidoctor/SampleIncludeProcessor.java

```diff
@@ -181,4 +182,37 @@ private String determineActiveTag(String line, List<String> tags) {
         }
         return null;
     }
+
+    private static String trimIndent(String source) {
+        String[] lines = source.split("\r\n|\r|\n");
+        int minIndent = Integer.MAX_VALUE;
+        for (String line : lines) {
+            if (line.trim().length() == 0) {
+                continue;
+            }
+
+            int indent = 0;
+            while (indent < line.length() && Character.isWhitespace(line.charAt(indent))) {
+                indent++;
+            }
+            if (indent < line.length()) {
+                minIndent = Math.min(minIndent, indent);
+            }
+        }
```

Comment:
could you put this in a method?
e.g. `calcMinIntent`

Reply:
Done